geode/pom.xml

```diff
@@ -45,8 +45,8 @@ LICENSE file.
   
   <repositories>
     <repository>
-      <id>gemstone</id>
-      <url>http://dist.gemstone.com.s3.amazonaws.com/maven/release</url>
+      <id>apache.snapshots</id>
+      <url>https://repository.apache.org/content/repositories/snapshots</url>
```

Comment:
please no snapshot repositories. if building currently requires the SNAPSHOT artifact, won't the current instructions for building from source put the needed bits in the user's local maven repo?